### PR TITLE
Migrate to GitHub Actions & Remove Babel Transpilation

### PR DESCRIPTION
**Pull request by mpyw**
# Summary

| Section | Previous | Now |
|:---|:---|:---|
| CI | Travis CI | GitHub Actions |
| Prettier | with ESLint | without ESLint |
| Transpiler | TSC + Babel | TSC |
| tslib | :x: | ✅ |

I gave up adopting rollup because it contains a huge dependency `@aws-sdk` which is not hosted on CDN.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,48 +4,47 @@
   "description": "Save your browser console logs to AWS CloudWatch (Inspired by agea/console-cloud-watch)",
   "main": "lib/index.js",
   "module": "es/index.js",
-  "types": "types/index.d.ts",
+  "types": "typings/index.d.ts",
```
Comment (mpyw) [inline package.json:7]: Renamed: `types` -> `typings`

File: package.json [hunk 2]
```diff
@@ -4,48 +4,47 @@
...
+  "sideEffects": false,
```
Comment (mpyw) [inline package.json:8]: For Tree Shaking

File: package.json [hunk 3]
```diff
@@ -4,48 +4,47 @@
...
   "keywords": [
     "aws",
     "cloudwatch",
     "logging",
     "browser"
   ],
   "dependencies": {
-    "@aws-sdk/client-cloudwatch-logs": "^3.10.0"
+    "@aws-sdk/client-cloudwatch-logs": "^3.10.0",
+    "tslib": "^2.3.0"
```
Comment (mpyw) [inline package.json:17]: `tslib`

File: src/index.ts [hunk 4]
```diff
@@ -1,4 +1,6 @@
-import Logger from "./Logger";
+import Logger from './logger';
```
Comment (mpyw) [inline src/index.ts:1]: Renamed: `Logger.ts` -> `logger.ts`

File: src/logger.ts [hunk 5]
```diff
@@ -146,9 +146,9 @@ export default class Logger {
 
     // Listen "error" event on window (type="uncaught")
     eventTarget.addEventListener(
-      "error",
-      async (error: any): Promise<void> => {
-        await this.onError(error, { type: "uncaught" });
+      'error',
+      async (error: unknown): Promise<void> => {
```
Comment (mpyw) [inline src/logger.ts:150]: ESLint

File: src/logger.ts [hunk 6]
```diff
@@ -162,7 +162,7 @@ export default class Logger {
    * @param e    - Error object
    * @param info - Extra Error Info (Consider using "type" field)
    */
-  public async onError(e: any, info?: ErrorInfo): Promise<void> {
+  public async onError(e: unknown, info?: ErrorInfo): Promise<void> {
```
Comment (mpyw) [inline src/logger.ts:165]: ESLint

File: src/logger.ts [hunk 7]
```diff
@@ -325,11 +325,11 @@ export default class Logger {
...
     // Cache fresh "logStreamName"
-    await this.setCache("logStreamName", logStreamName);
+    await this.setCache('logStreamName', logStreamName);
     return logStreamName;
   }
 
-  protected static isValidError<E = Error>(value: any): value is E {
-    return value && typeof value.message === "string";
+  protected static isValidError<E = Error>(value: unknown): value is E {
+    return Boolean(value && typeof (value as Error).message === 'string');
```
Comment (mpyw) [inline src/logger.ts:333]: ESLint

File: tests/stub.ts [hunk 8]
```diff
@@ -26,26 +26,26 @@ export class DummyStorage implements StorageInterface {
 }
 
 export interface DummyConsoleMessage {
-  message: string;
+  message: unknown;
```
Comment (mpyw) [inline tests/stub.ts:29]: ESLint

**Comment by mpyw** [comment]
Verified in Local Environment

**Review by hexium310 — APPROVED** [review]
It would be probably OK because the tests were passed.

**Pull request merged**